pkg/email/patch.go

```diff
@@ -68,3 +71,20 @@ func ParsePatch(text string) (title string, diff string, err error) {
 	}
 	return
 }
+
+func lineMatchesDiffStart(ln string) bool {
+	diffRegexps := []*regexp.Regexp{
+		regexp.MustCompile(`^(---|\+\+\+) [^\s]`),
+		regexp.MustCompile(`^diff --git`),
+		regexp.MustCompile(`^index [0-9a-f]+\.\.[0-9a-f]+`),
+		regexp.MustCompile(`^new file mode [0-9]+`),
+		regexp.MustCompile(`^Index: [^\s]`),
+		regexp.MustCompile(`^================`),
```

Comment:
Are these `==========` lines important? Can a patch be applied without them?
Usually they are not present. But I am worried that any line with a bunch of = will be recognized as patch start.

Reply:
Removed this from "diff start" markers and put into "diff continues" ones.